src/utils/find-files.ts

```diff
@@ -0,0 +1,31 @@
+import R from 'ramda'
+import path from 'path'
+import glob from 'glob'
+
+import { getFileExtensionPattern } from './read-file'
+
+export const isNotSchema = (filePath: string): boolean =>
```

Comment:
```suggestion
export const isSchema = (filePath: string): boolean =>
```

Reply:
Thanks for all the commit suggestions. When I hit this button now, it'll create a new commit with that small change? Guess we never `commit suggestion` but use this feature to make clear what changes we mean.

Reply:
updated

Reply:
maybe i got too excited about this feature. i actually really like this about github's review flow that you can make concrete code suggestions to save some feedback loops on smaller stuff.

however, i expected that it's possible to batch these together and _not_ create a new commit for ever single suggestion:

<img width="563" alt="image" src="https://user-images.githubusercontent.com/117495/66384041-4d36b600-e9be-11e9-87ce-3e81e880ba3e.png">

(thought that's what "Add suggestion to batch" was for)

Reply:
I saw that button too but i felt with a full-text search i'm quicker to update all this `describe('... behaves as expected')` spots.

I'll try it next time!